requirements.txt

```diff
@@ -6,10 +6,13 @@ notebook==6.5.3
 scikit-learn==1.2.2
 scipy==1.10.1
 tensorflow==2.11.1
-keras==2.12.0
+keras==2.11.0
```

Comment:
`pip` complained `tensorflow` 2.11 can only work with a `2.11` version of keras. Fine.